pmm/pmm2-api-tests.groovy

```diff
@@ -3,7 +3,7 @@ void runStaging(String DOCKER_VERSION, CLIENT_VERSION, CLIENTS) {
         string(name: 'DOCKER_VERSION', value: DOCKER_VERSION),
         string(name: 'CLIENT_VERSION', value: CLIENT_VERSION),
         string(name: 'PS_VERSION', value: '5.6'),
-        string(name: 'DOCKER_ENV_VARIABLE', value: '-e PMM_DEBUG=1 -e PERCONA_TEST_CHECKS_INTERVAL=10s -e PMM_RUN_STT_TESTS=1 -e PERCONA_TEST_AUTH_HOST=check-dev.percona.com:443'),
+        string(name: 'DOCKER_ENV_VARIABLE', value: '-e PMM_RUN_STT_TESTS=1 -e PMM_DEBUG=1 -e PERCONA_TEST_SAAS_HOST=check-dev.percona.com:443 -e PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX -e PERCONA_TEST_CHECKS_INTERVAL=10s -e PERCONA_TEST_IA=1 -e PERCONA_TEST_DBAAS=1'),
```

Comment:
Is it OK to use a hardcoded key here? I'd suggest using `withCredentials` and store it in Jenkins

Reply:
It is a public key

Reply:
I meant not from security purposes, but from easiness to change.

Reply:
I believe this way is easier for all of us, so we know what needs to be changed, so I don't mind